### PR TITLE
feat : added rules-page

### PR DESCRIPTION
**Pull request by Priya-Patel31**
It consists of :
* Header
* Main-container
 • Sidebar
 • Main-content-container
   1.Rules of quiz

### CODE REVIEW

File: rules_page/rules_page.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+.rules-page-image-container {
+  grid-column-start: 1;
+  grid-column-end: 2;
+}
+.quiz-details-container {
+  grid-column-start: 2;
+  grid-column-end: 3;
+}
+.quiz-info {
+  width: 6.25rem;
```
Comment (rkapoor10) [inline rules_page/rules_page.css:32]: maybe you will not need fixed widths if you are using grids

File: rules_page/rules_page.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+@media only screen and (max-width: 500px) {
+  .quiz-image-details-container {
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr auto;
+  }
+  .main-content-container {
+    margin: 1rem;
+  }
+}
```
Comment (rkapoor10) [inline rules_page/rules_page.css:49]: A suggestion - you may have just the search icon instead of the bar in mobile view.

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge. Go through the suggestions once.

**Review by t007rushi — APPROVED** [review]
Looks nice go ahead merge it

File: rules_page/rules_page.css (hunk 1 above)
Comment (Priya-Patel31) [reply rules_page/rules_page.css:32]: why we should not use fixed width in grids ??

**Pull request merged**